### PR TITLE
cli: add a note about submission key

### PR DESCRIPTION
**Pull request by pepyakin**
nonce handling is assumed to be perfomed only by a single process
in the network. Otherwise, there may be two processes that signed
an extrinsic with the same nonce. This may lead to rejection
of one of the extrinsics.

### CODE REVIEW

**Comment by pepyakin** [comment]
* **#271** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/271?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#270** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/270?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#269** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/269?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @pepyakin and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: ikura/shim/src/cli.rs [hunk 1]
```diff
@@ -48,6 +48,9 @@ pub struct KeyManagementParams {
     ///
     /// This key is enabled when running ikura-node in the local development mode.
     ///
+    /// NOTE: Submitting blobs is assumed to have complete control over the signing key. In case
+    /// there is another process running with the same key, it may cause error during submission.
```
Comment (gabriele-0201) [inline ikura/shim/src/cli.rs:52]: Wouldn't it be better to be more general here? Using 'another process' might mislead the user into thinking they can only run one shim processes with the same key. However, the assumption is stronger (please correct me if I'm wrong with [this]([url](https://github.com/thrumdev/blobs/pull/270#discussion_r1512477101)) example)

Comment (pepyakin) [reply ikura/shim/src/cli.rs:52]: The assumption is indeed stronger, but that's precisely what I was trying to convey by "another process". By it, I did not mean another ikura-shim process, and I did not even mean another another OS process, but I meant it in a more general sense: like as in "distributed process".

**Review by gab8i — APPROVED** [review]
Needs to be restacked. Besides that, I'm okay with how the user is informed to be careful with the provided key

**Pull request closed without merging**